model/team_test.go

```diff
@@ -21,45 +21,45 @@ func TestTeamJson(t *testing.T) {
 func TestTeamIsValid(t *testing.T) {
 	o := Team{}
 
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.Id = NewId()
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.CreateAt = GetMillis()
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.UpdateAt = GetMillis()
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.Email = strings.Repeat("01234567890", 20)
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.Email = "corey@hulen.com"
 	o.DisplayName = strings.Repeat("01234567890", 20)
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.DisplayName = "1234"
 	o.Name = "ZZZZZZZ"
-	if err := o.IsValid(); err == nil {
+	if err := o.IsValid(true); err == nil {
 		t.Fatal("should be invalid")
 	}
 
 	o.Name = "zzzzz"
 	o.Type = TEAM_OPEN
-	if err := o.IsValid(); err != nil {
+	if err := o.IsValid(true); err != nil {
```

Comment:
Wouldn't it be better to use `*utils.Cfg.TeamSettings.RestrictTeamNames` instead of `true`? Since if someone has `RestrictTeamNames` as false in the config all the unit tests will fail?